docs/en/integrations/language-clients/python/index.md

```diff
@@ -102,14 +102,14 @@ client = clickhouse_connect.get_client(host='localhost', username='default', pas
 ##### Use a ClickHouse Connect client instance to connect to a ClickHouse Cloud service:
 
 :::tip
-Use the connection details gathered earlier.  ClickHouse Cloud services require TLS, so use port 8443.
+Since ClickHouse Cloud services require TLS, add the `secure` flag and use port 8443.
```

Comment:
This is not actually true, clickhouse-connect will presume HTTPS if port 8443 is specified (it will also assume port 8443 if no port is specified and secure=True).

Reply:
weird I had received an exception without the flag, then after adding it I got it working and I guess I was somewhat biased since that was the fix for several of our language docs where the flag was missing.

however following your note, I've just omitted and that code snippet still executes fine, reverting this change.